byzcoin/service.go

```diff
@@ -1107,7 +1116,27 @@ func (s *Service) executeInstruction(cdbI CollectionView, cin []Coin, instr Inst
 	}
 	// Now we call the contract function with the data of the key.
 	log.Lvlf3("%s Calling contract %s", s.ServerIdentity(), contractID)
-	return contract(cdbI, instr, cin)
+	scs, cout, err = contract(cdbI, instr, cin)
+	if err != nil || instr.InstanceID == [32]byte{} {
+		// The config contract is ignore for the versioning
```

Comment:
should it read "the config contract ignores versioning"? or "the config contract doesn't have versioning"? But a contract doesn't have versions, its instance(s) have.
Why should the config-contract not have versions?

Reply:
I was under the impression that this particular contract was not stored but I just got confused myself. This part needed to handle unknown state changes when it has not been stored yet.